lib/semantics/pointer-assignment.cc

```diff
@@ -322,29 +345,93 @@ parser::Message *PointerAssignmentChecker::Say(A &&... x) {
   return msg;
 }
 
+static bool CheckPointerBounds(
```

Comment:
The function may deserve a small comment to explain that the returned `bool` does not indicate error/ok but bounds/bounds remapping.

Reply:
Good idea, I've done that.